webapp/channels/src/components/access_history_modal/access_history_modal.test.tsx

```diff
@@ -38,26 +40,23 @@ describe('components/AccessHistoryModal', () => {
         expect(wrapper.find(AuditTable).exists()).toBe(true);
     });
 
-    test('should have called actions.getUserAudits when onShow is called', () => {
+    test('should have called actions.getUserAudits only when first rendered', () => {
```

Comment:
Some tests descriptions may no longer apply, so we do our best to test the same and update the description accordingly.